spec/lib/artifact.spec.js

```diff
@@ -3,36 +3,37 @@
 describe("lib/artifact", () => {
     var Artifact, authTokenMock, bluebird, content, downloadLocation, error, fs, HttpLinkHeader, instance, logger, MetadataMock, requestMock, requestOptions, requestPromiseMock, responseHandler, responseMock, ShelfError, URI, uri;
 
-    authTokenMock = "abcd1234";
-    bluebird = require("bluebird");
-    downloadLocation = "example/download/path";
-    error = require("../../lib/error");
-    fs = require("fs");
-    logger = require("../../lib/logger")();
-    MetadataMock = require("../mock/metadata-mock")();
-    HttpLinkHeader = require("http-link-header");
-    requestMock = require("../mock/request-mock")();
-    requestOptions = require("../../lib/request-options")({
-        strictHostCheck: true
-    }, logger);
-    requestPromiseMock = require("../mock/request-promise-mock")();
-    responseMock = require("../mock/response-mock")();
-    ShelfError = require("../../lib/shelf-error")();
-    uri = "http://api.gisnep.example.com";
-    URI = require("urijs");
-    responseHandler = require("../../lib/response-handler")(bluebird, error, HttpLinkHeader, logger, ShelfError, URI);
-    Artifact = require("../../lib/artifact")(bluebird, fs, logger, requestMock, requestOptions, requestPromiseMock, responseHandler, MetadataMock);
     beforeEach(() => {
+        authTokenMock = "abcd1234";
```

Comment:
I know you weren't the one who named this, but technically this isn't a mock. Might wanna rename it to `testAuthToken` or something.